components/apimgt/org.wso2.carbon.apimgt.impl/src/test/java/org/wso2/carbon/apimgt/impl/APIGatewayManagerTest.java

```diff
@@ -617,7 +624,7 @@ public void testRemovingRESTAPIWithOutSequenceFromGateway() throws AxisFault {
         Map<String, String> failedEnvironmentsMap = gatewayManager
                 .publishToGateway(api, apiTemplateBuilder, tenantDomain);
         Assert.assertEquals(failedEnvironmentsMap.size(), 1);
-        Assert.assertTrue(failedEnvironmentsMap.keySet().contains(prodEnvironmentName));
+        Assert.assertTrue(failedEnvironmentsMap.keySet().contains(prodEnvironmentName ));
```

Comment:
Additional space at the end after prodEnvironmentName ?

Reply:
https://github.com/wso2/carbon-apimgt/pull/9546/commits/ad587f2f16bdef1c6b05054c42b780dda8e218b9